.github/workflows/ci_cd.yml

```diff
@@ -101,7 +101,19 @@ jobs:
           ANSYSLMD_LICENSE_FILE: 1055@${{ secrets.LICENSE_SERVER }}
           ANSYS_WORKBENCH_LOGGING_FILTER_LEVEL: 0
         run: |
-          xvfb-run /install/ansys_inc/v232/aisol/.workbench_lite make -C doc html
+          xvfb-run /install/ansys_inc/v232/aisol/.workbench_lite make -C doc html > doc_build_ouput.txt
+          cat doc_build_ouput.txt
+          #
+          # Check if Error occurred
+          #
+          output=$(grep -c "Error" doc_build_ouput || true)
```

Comment:
"Error" alone might be too generic... Search for a regex involving ```make: .* Error .*``` Validate the regex offline first. We have to check for errors on the make process